curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -167,7 +169,24 @@ async (getUserInput) => {
     body: `username=fcc_test_${Date.now()}`.substr(0, 29)
   });
   if (res.ok) {
-    assert(true);
```

Comment:
Here, the check I made here was only that the user can create a new user, while I should test if he can create a new exercise